packages/@uppy/aws-s3-multipart/src/index.ts

```diff
@@ -752,6 +752,19 @@ export default class AwsS3Multipart<
         const etag = xhr.getResponseHeader('ETag')
         const location = xhr.getResponseHeader('Location')
```

Comment:
I think we can get rid of those, no?

Reply:
It's still used for the if statements to log warnings.

Reply:
```suggestion
        const {etag, location} = headersMap
```
This needs to be moved below